hugo/layouts/partials/copy_to_clipboard.html

```diff
@@ -0,0 +1,4 @@
+{{ if (findRE "<pre" .Content 1) }}
+    {{ $clipboardJS := resources.Get "js/clipboard.js" }}
+    <script>{{ $clipboardJS.Content | safeJS }}</script>
```

Comment:
Why inlining the script rather than `<script src="{{ $clipboardJS.RelPermalink }}"/>` ?

Reply:
Fair point, I've added this now and fingerprinted it to handle caching (not that we probably expect this file to update much). Commit 57e7b0936a239158e156cbc775ace11e22898d31